README.md

```diff
@@ -22,13 +22,13 @@ authenticator on your Kubernetes master using host networking so that the
 apiserver can access the authenticator through the loopback interface.
 
 ```
-kubectl create -f https://github.com/oursky/kubernetes-github-authn/blob/master/manifests/github-authn.yaml
+kubectl create -f https://raw.github.com/oursky/kubernetes-github-authn/blob/master/manifests/github-authn.yaml
```

Comment:
Thanks for spotting out the mistake!! @agonzalezro

I think the link should start with `https://raw.githubusercontent.com` instead of `https://raw.github.com`. Let me help change it :)

Reply:
It redirects from one to the other. Take a look because both are returning 404 to me at the moment.

Thanks!

Reply:
Thank you @agonzalezro :D